great_expectations/data_context/store/store_backend.py

```diff
@@ -413,3 +413,83 @@ def has_key(self, key):
         all_keys = self.list_keys()
         return key in all_keys
 
+
+class FixedLengthTupleGCSStoreBackend(FixedLengthTupleStoreBackend):
+    """
+    Uses a GCS bucket as a store.
+
+    The key to this StoreBackend must be a tuple with fixed length equal to key_length.
+    The filepath_template is a string template used to convert the key to a filepath.
+    There's a bit of regex magic in _convert_filepath_to_key that reverses this process,
+    so that we can write AND read using filenames as keys.
+    """
+    def __init__(
+        self,
+        root_directory,
+        filepath_template,
+        key_length,
+        bucket,
+        prefix,
+        project,
+        forbidden_substrings=["/", "\\"],
```

Comment:
note to self: harmonize this with the change for platform independent paths in #848